scripts/config/abora-config-gui.py

```
@@ -0,0 +1,407 @@
+
+    def _apply_worker(self, changed: dict[str, str]):
+        # pkexec/sudo don't reliably pass through the calling process's
+        # environment, so ABORA_SYSTEM_CONFIG has to ride inside the
+        # escalated command itself (via `env`) -- otherwise abora-config.sh
+        # would silently fall back to its own default (/etc/nixos) instead
+        # of whatever config dir this GUI was actually pointed at (e.g. a
+        # throwaway test fixture), which could mean writing to the real
+        # system by accident during testing.
+        prefix = sudo_prefix() + ['env', f'ABORA_SYSTEM_CONFIG={CONFIG_DIR}']
+        for key, value in changed.items():
+            proc = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)